CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v4.2.4](https://github.com/puppetlabs/puppet-lint/tree/v4.2.4) - 2024-02-09
+
+[Full Changelog](https://github.com/puppetlabs/puppet-lint/compare/v4.2.3...v4.2.4)
+
+### Fixed
+
+- (GH-189) - invalid fact correction for top scope structured fact [#190](https://github.com/puppetlabs/puppet-lint/pull/190) ([jordanbreen28](https://github.com/jordanbreen28))
+- (CAT-1688) Upgrade Rubocop to `~> 1.50.0` [#185](https://github.com/puppetlabs/puppet-lint/pull/185) ([LukasAud](https://github.com/LukasAud))
```

Comment:
not sure if that should be listed under `Fixed`

Reply:
yeah agreed, bit of a pain point with the changelog generator, if we label as maintenance it won't show on the changelog. 
That being said, I do think the maintenance label is more appropriate in this case.